### PR TITLE
chore: update stale issue workflow

### PR DESCRIPTION
**Pull request by peterwoodworth**
*Description of changes:* redo of https://github.com/aws/aws-sdk/pull/400 but with signed commits, and with updated ancient time

closes #400, closes #462

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by aBurmeseDev — COMMENTED** [review]
Added single comment on stale-issue-cleanup version.

File: .github/workflows/close-stale-issues.yml [hunk 1]
```diff
@@ -1,30 +1,51 @@
...
   cleanup:
+    permissions:
+      issues: write
+      contents: read
+      pull-requests: write
     runs-on: ubuntu-latest
     name: Stale issue job
     steps:
-    - uses: aws-actions/stale-issue-cleanup@v3
+    - uses: aws-actions/stale-issue-cleanup@v5
```
[the remark this answers is not in the document]
Comment (peterwoodworth) [reply .github/workflows/close-stale-issues.yml]: Done

**Comment by peterwoodworth** [comment]
New changes in commit: Removed the ancient message prop, removed feature-request from exempt labels (not necessary with ancient message disabled), and bumped to v6

**Review by aBurmeseDev — APPROVED** [review]
🚀

**Pull request merged**